assets/notebooks/assembly_report.qmd

```diff
@@ -164,14 +164,82 @@ We can glean a good amount of information about the sequenced species from the k
-of peaks in the k-mer histogram. A haploid organism should have one peak, while a diploid should have two peaks.
+
+::: {.callout-note title="what is GenomeScope and how to interpret the GenomeScope plot" collapse="true"}
+#### **GenomeScope Overview**
+
+**GenomeScope** is a computational tool designed to analyze raw sequencing data and provide insights into the genome's characteristics, such as its **size**, **heterozygosity rate**, and **repeat content**. It works by analyzing **k-mer frequency distributions** derived from sequencing reads and fitting a mathematical model to these distributions.
+
+---
+
+##### **How GenomeScope Works**
+
+1. **Input Data**  
+   GenomeScope uses a **k-mer frequency histogram** generated by tools like **Jellyfish**.  
+   - A **k-mer** is a sequence of *k* nucleotides.  
+   - The histogram represents how often each unique k-mer appears in the sequencing data.
+
+2. **Model Fitting**  
+   GenomeScope fits a **mixture model of negative binomial distributions** to the k-mer histogram. This model accounts for:  
+   - **Unique sequences**  
+   - **Repeated sequences**  
+   - **Heterozygous and homozygous regions**
+
+3. **Output**  
+   The tool estimates:  
+   - **Genome size**  
+   - **Heterozygosity**  
+   - **Repeat content**  
+   It also generates **plots** and **tables** summarizing these properties.
+
+---
+
+##### **Interpreting the GenomeScope Plot**
+
+The plot typically includes:
+
+- **X-axis**: Coverage (how many times each k-mer appears in the data).  
+- **Y-axis**: Frequency (number of k-mers at each coverage level).  
+
+###### **Peaks**  
+- The **first peak** (lower coverage) corresponds to **heterozygous regions**.  
+- The **second peak** (higher coverage) corresponds to **homozygous regions**.  
+- Additional peaks at higher coverage levels represent **repetitive sequences**.
+
+###### **Fit Curve**  
+- A line representing the **model fitted to the data**.
+
+---
+
+###### **Key Insights from the Plot**
+
+1. **Genome Size**  
+   - Calculated by normalizing the total number of k-mers based on the coverage of homozygous sequences.
+
+2. **Heterozygosity**  
+   - Determined by comparing the heights of the heterozygous and homozygous peaks.  
+   - A **higher first peak** indicates greater heterozygosity.
+
+3. **Repeat Content**  
+   - Represented by additional peaks at higher coverage levels.  
+   - **More pronounced peaks** indicate higher repeat content.
+
+---
+
+##### **Example Interpretation**
+
+###### **For a Diploid Genome**:  
+- A **bimodal distribution** is expected, with two main peaks for heterozygous and homozygous regions.  
+- If the **first peak** is high relative to the second, it suggests **high heterozygosity**.  
+- If there are significant peaks at very high coverage, it indicates **repetitive elements** in the genome.
+
+###### **For a Haploid Organism**:  
+- A single peak is expected.
```

Comment:
We need pictures here. I think we need a link out to workshop material or something.